2025-08-28 20:56:00,076 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2025-08-28 20:56:00,076 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/hawkerd/projects/mcp-server-demo/.venv/lib/python3.10/site-packages/instabot/bot/bot.py
2025-08-28 20:56:00,077 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2025-08-28 20:56:00,285 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2025-08-28 20:56:00,508 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2025-08-28 20:56:00,805 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2025-08-28 20:56:01,053 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Sorry, there was a problem with your request.","status":"fail","error_type":"unexpected_login_error"}'
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: Sorry, there was a problem with your request.
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - INFO - Error type: unexpected_login_error
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2025-08-28 20:56:06,738 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
